crates/sui-types/src/metrics.rs

```diff
@@ -136,3 +136,29 @@ impl BytecodeVerifierMetrics {
         }
     }
 }
+
+pub struct SignatureMetrics {
+    /// Count of zklogin signatures
+    pub zklogin_sig_count: IntCounter,
+    /// Count of multisig signatures
+    pub multisig_sig_count: IntCounter,
```

Comment:
i would probably just roll these into AuthorityMetrics, but its up to you

Reply:
revised!